packages/compiler/lib/lib.cadl

```diff
@@ -31,10 +31,13 @@ model Array<T> {}
 model Record<T> {}
 
 /**
- * Represent a URI string.
+ * Represent a URI string as described by https://url.spec.whatwg.org/#relative-url-string.
  */
-@format("uri")
-scalar uri extends string;
+@format("url")
```

Comment:
Do we want to change the format to `url`. See also #1329  for removing `@format` decorator

Reply:
Well, we need cadl-autorest to continue to emit `"format": "uri"`, or we need to change autorest.   

I would be for making this format value change and adding a translation of 'format' values in cadl-autorest (I don't think we should expect a 1:1 correspondence between Cadl and Swagger on these things, where it doesn't make sense).

We will also want to add a discussion of url and url encoding in the documentation (and we want to add this to the breakign change issue)

Reply:
openapi and autorest don't actually make use of the format decorator for `uri` since the scalar change(which is why I think we should just remove `@format` or at least move it to openapi)

Reply:
I would need to make the same change in autorest as done here in openapi3 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/1031227/205184143-0583c75c-8334-46b3-bd21-ca9836485d27.png">